packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -159,6 +167,14 @@ export class DataSyncPlugin extends GraphbackPlugin {
 
       modelTC.addFields(metadataFields);
 
+      if (this.conflictEngine !== undefined) {
```

Comment:
As mentioned. Always add versioned and only override if someone provided something else.

Reply:
Please check how current offix does it. We do not want to downgrade capabilities unless we will have reason for it.

Reply:
So do we always have conflicts?

Reply:
Or do we always have version but conflict is optional?

Reply:
It all depends how we structure API. 
I was thinking that conflict stuff has different helper and uses versioned out of the box, but if you want to have one helper then we need to pass version. 

Generally reason why we want to have conflict separate is that it will more or less be less performant way to edit stuff than without it. So if you do not use conflict you do not want to be affected by conflict shenanigans. 

As mentioned all of this is pretty much open to your interpretation, but the way we have it now suggest that we getting the worse of both worlds:

- Mixed implementation
- Increased complexity
- Verbose API
- Lack of clear distinction between conflict and non conflicts.